src/wallet/rpcwallet.cpp

```diff
@@ -187,6 +187,11 @@ UniValue getnewaddress(const JSONRPCRequest& request)
     return EncodeDestination(dest);
 }
 
+void DeleteLabel(CWallet& wallet, std::string label)
```

Comment:
Nit, move to `CWallet::DeleteLabel(const std::string& label)`? Or make it `static` for now?

Reply:
Agree, this would be the only direct `CWalletDB` call from rpcwallet, seems as though it should go through `CWallet`

Reply:
Moved into `CWallet`